src/main/resources/templates/fragments/topnav.html

```diff
@@ -1,17 +1,21 @@
 <div th:fragment="navbar" xmlns:th="http://www.w3.org/1999/xhtml">
-  <nav class="navbar navbar-expand-lg navbar-light bg-light">
-    <h2> Meal Mentor </h2>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
+    <style>
+        h2 {
+            color: #2faf9f;
+        }
+
+        li {
+            padding-left: 70px;
+        }
+    </style>
```

Comment:
I noticed some of the styling for the navbar was in the pages instead of the navbar itself. Save yourself some time and only do it once.

Reply:
I recommend going one step further: export this to a linked sytlesheet.